ui/app/src/utils/hooks.ts

```diff
@@ -74,7 +74,7 @@ export function createResource<T = any>(factoryFn: () => Promise<T>) {
   return resource;
```

Comment:
Remove this file from your commit. There are a few updates that will take place here.